docs/device-management/5-ota/README.md

```diff
@@ -4,33 +4,75 @@ title: Over-the-Air (OTA) Updates Overview
 slug: /device-management/ota
 ---
 
-Golioth Device Firmware Update (DFU) enables quick, secure deployment of firmware packages to IoT devices throughout the world. The Golioth Web Console enables easy management of firmware releases, including multi-part binary bundles, enabling updates for devices as diverse as smart speakers, digital signage, machine learning enabled sensor systems, multiple processor embedded devices, and more.
+The Golioth Over-the-Air (OTA) firmware update service enables quick, secure
+deployment of firmware packages to IoT devices throughout the world. The Golioth
+Web Console enables easy management of firmware releases, including multi-part
```

Comment:
nitpick: given that the web console is one interface to the Golioth API, I would lean towards using "Golioth API" here.
```suggestion
deployment of firmware packages to IoT devices throughout the world. The Golioth
API enables easy management of firmware releases, including multi-part
```